### PR TITLE
[test] Remove unnecessary tags in the HaloExchangeTest 

### PR DESCRIPTION
**Pull request by kraushm**
Based on Jira issue https://jira.cscs.ch/browse/UES-772

### CODE REVIEW

File: cscs-checks/mch/collectives_halo.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def __init__(self, variant, bench_reference):
         }
 
         self.maintainers = ['AJ', 'MKr']
-        self.tags = {'production', 'mch', 'craype'}
+        self.tags = {'mch'}
```
Comment (vkarak) [inline cscs-checks/mch/collectives_halo.py]: The tags should be removed only for Tsa/Arolla, right?

Comment (kraushm) [reply cscs-checks/mch/collectives_halo.py]: Indeed, my bad.

**Comment by vkarak** [comment]
@jenkins-cscs retry daint

**Comment by vkarak** [comment]
@jenkins-cscs retry none

**Pull request merged**